spec/models/tribunal_case/tribunal_case_purge_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+RSpec.describe TribunalCase, '.purge!', type: :model do
+  subject { described_class }
+  let(:cases) { class_double(described_class) }
+
+  before do
+    allow(subject).to receive(:where).and_return(cases)
+    allow(cases).to receive(:delete_all)
+    subject.purge!('2017-01-01')
+  end
+
+  it 'finds cases created before a specified date' do
+    expect(subject).to have_received(:where).with(['created_at < ?', '2017-01-01'])
```

Comment:
Isn't this rather expecting the thing we're supposed to be testing? 

How about having two cases in the system - one older and one younger than N days, and then confirming that only the younger case remains after the call to "purge!"?